qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/resolvers/characterescapes/SafeValueResolverTest.java

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+
+package com.redhat.qute.resolvers.characterescapes;
+
+import static com.redhat.qute.QuteAssert.assertHover;
+import static com.redhat.qute.QuteAssert.c;
+import static com.redhat.qute.QuteAssert.r;
+import static com.redhat.qute.QuteAssert.testCompletionFor;
+import static com.redhat.qute.QuteAssert.testDiagnosticsFor;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for 'raw' value resolver.
+ *
+ * @see https://quarkus.io/guides/qute-reference#character-escapes
+ *
+ */
+public class SafeValueResolverTest {
+	@Test
+	public void diagnostics() throws Exception {
+		String template = "{@org.acme.Item item}\r\n" + //
+				"{item.name.safe}";
+		testDiagnosticsFor(template);
+
+		template = "{@org.acme.Item item}\r\n" + //
```

Comment:
do the same thing than raw (add a new test diagnostics + movethe test in completion).